### PR TITLE
[hi] Update hindi

### PR DESCRIPTION
**Pull request by nishadil**
[hi] Update hindi

### CODE REVIEW

File: locales/hi/php-inline.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "accepted": "इस क्षेत्र को स्वीकार किया जाना चाहिए ।",
-    "accepted_if": "This field must be accepted when :other is :value.",
+    "accepted_if": ":Attribute को ज़रूर स्वीकार किया जाना चाहिए जब :other :value हो ।",
```
Comment (andrey-helldar) [inline locales/hi/php-inline.json:3]: In the inline translation file, you do not need to access the attribute.

For example:

> Main: The :attribute must be accepted when :other is :value.
> Inline: This field must be accepted when :other is :value.

File: locales/hi/php-inline.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "date_equals": "यह :date के बराबर तारीख होनी चाहिए ।",
     "date_format": "यह प्रारूप :format से मेल नहीं खाता है ।",
     "declined": "This value must be declined.",
-    "declined_if": "This value must be declined when :other is :value.",
+    "declined_if": ":Attribute को ज़रूर अस्वीकार किया जाना चाहिए जब :other :value हो ।",
```
Comment (andrey-helldar) [inline locales/hi/php-inline.json:24]: In the inline translation file, you do not need to access the attribute.

For example:

> Main: The :attribute must be declined when :other is :value.
> Inline: This field must be declined when :other is :value.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]
Is it true that some sentences have a space before the `|` and some do not?

For example:

```
"accepted_if": ":Attribute को ज़रूर स्वीकार किया जाना चाहिए जब :other :value हो ।",
"mac_address": "इस फ़ील्ड में एक मान्य MAC address होना चाहिए।"
```

**Comment by andrey-helldar** [comment]
If you want to add new changes to a PR, you don't need to close the old one and create a new one PR. It is enough to create a commit and execute the `git push` command. All new changes will go into the previously created PR, provided the changes are made to the same branch.

**Comment by nishadil** [comment]
> Is it true that some sentences have a space before the `|` and some do not?
> 
> For example:
> 
> ```
> "accepted_if": ":Attribute को ज़रूर स्वीकार किया जाना चाहिए जब :other :value हो ।",
> "mac_address": "इस फ़ील्ड में एक मान्य MAC address होना चाहिए।"
> ```

Actually, in Hindi language symbol `।` used in two places and meaning of it depends on where we used it. like "होना", "बड़ा" etc ends with similar symbol while these are complete words, so at the end of sentences we mostly use some space between last word and sentences closer `।` to avoid creating any reading illusion.

**Pull request merged**